packages/shopify-api/lib/auth/oauth/safe-compare.ts

```diff
@@ -1,5 +1,10 @@
 import * as ShopifyErrors from '../../error';
 
+export type SafeCompare = (
+  strA: string | {[key: string]: string} | string[] | number[],
+  strB: string | {[key: string]: string} | string[] | number[],
+) => boolean;
+
 export function safeCompare(
```

Comment:
Total nit, but we could export this as an arrow function so we don't have to repeat the types:

```suggestion
export const safeCompare: SafeCompare = (strA, strB) => {
```